src/SignUp/SignInScreen.js

```diff
@@ -0,0 +1,464 @@
+    width: '100%',
+    fontSize: 18,
+    marginLeft: 60,
+  },
+  text: {
+    marginLeft: 17,
+    marginTop: 5,
+    width: 330,
+    margin: 8.5,
+    flexDirection: 'row',
+    alignItems: 'center',
+    textAlignVertical: 'center',
+    textAlign: 'center',
+  },
+});
+
+export default function SignUpScreen({ navigation }) {
+  // first page states:
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [organization, setOrganization] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [number, setNumber] = useState('');
+  const [busPhone, setBusPhone] = useState('');
+  const [hidePass, setHidePass] = useState(true);
+  const [hidePassConf, setHidePassConf] = useState(true);
+
+  const lastNameInput = useRef();
+  const organizationInput = useRef();
+  const emailInput = useRef();
+  const passwordInput = useRef();
+
+  const confirmPasswordInput = useRef();
+  const numberInput = useRef();
+  const busPhoneInput = useRef();
+
+  // second page states:
+  const [recipient, setRecipient] = useState('');
+  const [address, setAddress] = useState('');
+  const [apt, setApt] = useState('');
+  const [zip, setZip] = useState('');
+  const [instr, setInstr] = useState('');
+
+  const addrInput = useRef();
+  const aptInput = useRef();
+  const zipInput = useRef();
+  const instrInput = useRef();
+
+  const [page1, setPage1] = useState(true);
+
+  const checkInputs = () => {
+    if (firstName === '') {
+      Alert.alert('Please enter the First Name to proceed');
+    } else if (lastName === '') {
+      Alert.alert('Please enter the Last Name to proceed');
+    } else if (email === '') {
+      Alert.alert('Please enter the Email to proceed');
+    } else if (password === '') {
+      Alert.alert('Please enter the Password to proceed');
+    } else if (number === '') {
+      Alert.alert('Please enter the Personal phone number to proceed');
+    } else if (password !== confirmPassword) {
+      Alert.alert('Password Confirmation does not match Password');
+    } else {
+      return true;
+    }
+    return false;
+  };
+
+  const handleSignUp = () => {
+    if (address === '') {
+      Alert.alert('Please enter the Address to proceed');
+    } else if (zip === '' || isNaN(zip)) {
+      Alert.alert('Please enter the Zipcode to proceed');
+    } else {
+      base('Users').create([
+        {
+          fields: {
+            email,
+            password,
+            'first name': firstName,
+            'last name': lastName,
+            organization,
+            'personal phone': number,
+            'business phone': busPhone,
+            recipient,
+            address,
+            apt,
+            zip,
+            instr,
+          },
+        },
+      ], (err) => {
+        if (err) {
+          Alert.alert(err, err.message);
```

Comment:
Change this to Alert.alert(err.error, err.message) to get rid of the ReadableNativeMap error